src/core/geometry/qgsgeometry.h

```diff
@@ -1700,7 +1700,7 @@ class CORE_EXPORT QgsGeometry
      *
      * \since QGIS 3.0
      */
-    QgsGeometry subdivide( int maxNodes = 256 ) const;
+    QgsGeometry subdivide( int maxNodes = 256, double gridSize = -1 ) const;
```

Comment:
The gridSize arguments need documentation, including a "since QGIS 3.24" and a description of what they mean